icaruscode/CRT/CRTDecoder/DecoderICARUSCRT_module.cc

```diff
@@ -407,16 +421,25 @@ void crt::DecoderICARUSCRT::produce(art::Event& evt)
         destCh += recipe.direction; // increase or decrease the source
 
       }
-
+      allCRTdata.push_back(data);
     } // for all recipes
 
   } // for all input data
-
+  std::cout << "allCRTdata.size(): " << allCRTdata.size() << "\n";
```

Comment:
I _recommend_ this output to be moved to a debug stream or removed:
```suggestion
  mf::LogTrace{ "DecoderICARUSCRT" } << "allCRTdata.size(): " << allCRTdata.size();
```